Backends/CLX/medium.lisp

```diff
@@ -406,10 +406,19 @@ translated, so they begin at different position than [0,0])."))
       (push #'(lambda () (xlib:free-pixmap mm)) ^cleanup)
       mm)))
 
+;;; For more details, see also: https://github.com/sharplispers/clx/pull/146
+(defun cached-drawable-depth (drawable)
+  "The purpose of this is to reduce local network traffic for the case of many
+calls to compute-rgb-image, for example when drawing a pattern."
```

Comment:
make this docstring a comment and rename the function to `ensure-drawable-depth`

Reply:
I moved the docstring. We discussed the naming on IRC.